Makefile.docker

```diff
@@ -0,0 +1,15 @@
+DOCKER_IMAGE ?= fedora:latest fedora:rawhide
+
+# escaping colon is broken in older versions of make,
+# so rather replace it with placeholder
+TEST_IMAGE := $(subst :,_COLON_,$(DOCKER_IMAGE))
+
+.PHONY: test $(TEST_IMAGE)
+
+test: $(TEST_IMAGE)
+
+$(TEST_IMAGE):
+	$(eval IMAGE := $(subst _COLON_,:,$@))
+	sudo docker pull $(IMAGE)
```

Comment:
Ok, I'll add check for docker group and call sudo only in case it's really needed.